flask_login.py

```diff
@@ -385,16 +385,10 @@ def _session_protection(self):
         app = current_app._get_current_object()
         mode = app.config.get('SESSION_PROTECTION', self.session_protection)
 
-        # if there is no '_id', then take the current one for good
-        if '_id' not in sess:
-            sess['_id'] = ident
+        # if the sess is empty, it's an anonymous user or just logged out
```

Comment:
This comment is out of context since the code it references is removed, so this comment shouldn't be here.